### PR TITLE
fixed failing tests

### PR DESCRIPTION
**Pull request by bancer**

### CODE REVIEW

**Comment by dereuromark** [comment]
Do you know how we can get 7.2 to run green? different phpunit version?

**Comment by bancer** [comment]
It is hard to say without knowledge of the content of `../travis/script.sh` file.

**Comment by bancer** [comment]
@dereuromark Is there anything that stops this from being merged?

File: Test/Case/Model/ShimModelTest.php [hunk 1]
```diff
@@ -291,6 +296,9 @@ public function testFindRecursive() {
 	 * Testing missing contain warnings
 	 *
 	 * @expectedException PHPUnit_Framework_Error_Warning
+	 * @expectedExceptionMessage No recursive -1 or contain used for the query in User: Array
+(
```
Comment (dereuromark) [inline Test/Case/Model/ShimModelTest.php]: Looks not good.

**Comment by dereuromark** [comment]
Tests are not even running right now, we might have to really copy the https://github.com/FriendsOfCake/travis/blob/master/before_script.sh etc over.

File: Test/Case/Model/ShimModelTest.php (hunk 1 above)
Comment (bancer) [reply Test/Case/Model/ShimModelTest.php]: That's what the message is. Placing the `()` to the line above makes the test to fail when run with PHPUnit 5.7.

Comment (dereuromark) [reply Test/Case/Model/ShimModelTest.php]: Why? does it need always exp message?

Comment (dereuromark) [reply Test/Case/Model/ShimModelTest.php]: I dont think so, if you look into their docs, expectedExceptionMessage is always optional.

Comment (bancer) [reply Test/Case/Model/ShimModelTest.php]: screenshot from 2018-04-18 16 31 09

Comment (bancer) [reply Test/Case/Model/ShimModelTest.php]: I didn't find the way to make it optional https://phpunit.de/manual/5.7/en/appendixes.annotations.html#appendixes.annotations.expectedExceptionMessage.

Comment (bancer) [reply Test/Case/Model/ShimModelTest.php]: :man_facepalming: `@expectedExceptionMessage No recursive -1 or contain used for the query in User` should be actually fine.

**Comment by dereuromark** [comment]
We can try to fix the travis repo: https://github.com/FriendsOfCake/travis/pull/75
That might also help with the other runs.

**Pull request merged**